### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by frederikprijck**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/c/sdks/5
+    about: Discuss this SDK in the Auth0 Community forums
+  - name: SDK API Documentation
+    url: https://auth0.github.io/auth0.net/
+    about: Read the API documentation for this SDK
```
Comment (frederikprijck) [inline .github/ISSUE_TEMPLATE/config.yml:8]: This repository doesn't have a FAQ or Library documentation, so I didn't include those.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/c/sdks/5
```
Comment (frederikprijck) [inline .github/ISSUE_TEMPLATE/config.yml:4]: I couldn't find a tag specific for auth0.net on the community site.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest an idea or a feature for this project
+title: ''
+labels: feature request
+assignees: ''
+
+---
+
+**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
```
Comment (Widcket) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/responsible-disclosure-policy) details the procedure for disclosing security issues.
```

**Review by Widcket — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (frederikprijck) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Updated

**Review by Widcket — APPROVED** [review]

**Review by Widcket — APPROVED** [review]

**Pull request merged**